flowmachine/tests/test_subscriber_handset_stats.py

```diff
@@ -19,7 +19,6 @@
         ("width", "avg", "JZoaw2jzvK2QMKYX", 28.0178288001543),
         ("width", "max", "DELmRj9Vvl346G50", 45.9),
         ("width", "min", "9vXy462Ej8V1kpWl", 5.19),
-        ("width", "median", "KXVqP6JyVDGzQa3b", 17.1),
```

Comment:
As noted above, I think this actually is valid (it was the presence of this test, and the fact it hasn't been failing, that made me go back and look at the query implementation more closely).